src/components/Dialogue/Dialogue.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+import Dialogue from './Dialogue';
+import { DialogTitle } from './Children/DialogTitle';
+import { DialogList } from './Children/DialogList';
+import { ListItem } from './Children/ListItem';
+import { DialogActions } from './Children/DialogActions';
+import { DialogContent } from './Children/DialogContent';
+import { DialogContentInfo } from './Children/DialogContentInfo';
```

Comment:
like the way you separate the components, saw other people putting everything into one component

Reply:
Thank you!